<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Medium</family>
      <pointsize>15</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>кофе)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>поиск!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>2</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Noto Sans Medium</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Любой тип</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="any_type">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Noto Sans Medium</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Молотый</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="molotiy">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Noto Sans Medium</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>В зернах</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="zerna">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Noto Sans Medium</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Цена(до):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="price_limit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>140</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_button">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
